README.md

```diff
@@ -9,12 +9,17 @@ app shows a basic example of using this library.
 
 # Table of Contents
 
-- [Installation](#Installation)
-- [Getting Started](#Getting-Started)
-- [Example](#Basic-Example)
-- [Documentation](#Documentation)
-  - [State](#State)
-  - [Actions](#Actions)
+- [Yext Answers: React](#yext-answers-react)
```

Comment:
Nit:
```suggestion
- [Yext Search: React](#yext-answers-react)
```

Reply:
Can you update the other instances too? And rename the repo? And update any client repos using this repo to use `search` instead of `answers`?